addon/components/pix-check-box.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="pix-check-box pix-check-box--grey {{if this.isChecked "pix-check-box--blue"}}">
```

Comment:
**suggestion** : plutôt que de fonctionner avec des modifiers de couleur, plutôt opter pour des modifiers d'état : `{{if this.isChecked "pix-check-box--checked"}}`

Reply:
Quand la check-box est checked elle a les 2 classes: `blue` et `grey`. Essaye plutôt d'avoir qu'une seule classe qui  s'adapte en fonction de la couleur (ou de l'état si tu changes après le commentaire de Lise)

Reply:
**suggestion**
Il me semble que "checkbox" est en un seul morceau, non ?
Si oui, est-ce qu'on pourrait avoir le nommage `pix-checkbox` du coup plutôt que `pix-check-box` ? (Dans le code et noms des fichiers du coup 😅 )

Reply:
ce sera plus checkbox, ça va être renommé @Anne-Gaelle-S